types/three/src/renderers/shaders/UniformsUtils.d.ts

```diff
@@ -1,7 +1,8 @@
 import { UniformsGroup } from '../../core/UniformsGroup.js';
+import { IUniform } from './UniformsLib.js';
 
-export function cloneUniforms(uniforms_src: any): any;
-export function mergeUniforms(uniforms: any): any;
+export function cloneUniforms(uniformsSrc: { [uniform: string]: IUniform }): { [uniform: string]: IUniform };
```

Comment:
I think making this generic will make this better for users of this function:
```suggestion
export function cloneUniforms<T extends { [uniform: string]: IUniform }>(uniformsSrc: T): T;
```